I tried to specify import path with non-standard port, but it does not compile.

```
$ cat main.go
package main

import foo "localhost:61000/a/b"

func main() {
        foo.Foo()
}
$ go build
can't load package: package github.com/hnakamur/tmp: 
main.go:3:12: invalid import path: "localhost:61000/a/b"
$ go version
go version go1.7rc3 linux/amd64
```

I saw [git - How Can i specify ports using golang Remote import paths? - Stack Overflow](http://stackoverflow.com/questions/34922929/how-can-i-specify-ports-using-golang-remote-import-paths) and it does not seem we can specify import path with non-standard port.

As for bankd to serving http connections, is there an example?

--- a/imports.go
+++ b/imports.go
@@ -99,6 +99,10 @@ func parseImports(recursive bool) ([]string, error) {
 				continue
 			}
 
+			if inTests {
+				importpath = strings.Replace(importpath, "__blankd__", "localhost:60001", -1)
+			}
+
 			if isOwnPackage(importpath, cwd) {
 				continue
 			}

--- a/main.go
+++ b/main.go
@@ -1,8 +1,11 @@
 package main
 
 import (
+	"crypto/tls"
 	"fmt"
 	"log"
+	"net/http"
+	"os"
 	"strconv"
 	"strings"
 
@@ -46,7 +49,24 @@ Options:
 
 var (
 	verbose bool
+	inTests bool
 )
+
+func init() {
+	for i, arg := range os.Args {
+		if arg == "--integration-test" {
+			inTests = true
+			os.Args = append(os.Args[:i], os.Args[i+1:]...)
+		} else if arg == "--insecure-skip-verify" {
+			http.DefaultClient.Transport = &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
+				},
+			}
+			os.Args = append(os.Args[:i], os.Args[i+1:]...)
+		}
+	}
+}
 
 func main() {
 	args := godocs.MustParse(usage, version)

--- a/main.go
+++ b/main.go
@@ -53,19 +53,21 @@ var (
 )
 
 func init() {
-	for i, arg := range os.Args {
+	newArgs := make([]string, 0, len(os.Args))
+	for _, arg := range os.Args {
 		if arg == "--integration-test" {
 			inTests = true
-			os.Args = append(os.Args[:i], os.Args[i+1:]...)
 		} else if arg == "--insecure-skip-verify" {
 			http.DefaultClient.Transport = &http.Transport{
 				TLSClientConfig: &tls.Config{
 					InsecureSkipVerify: true,
 				},
 			}
-			os.Args = append(os.Args[:i], os.Args[i+1:]...)
+		} else {
+			newArgs = append(newArgs, arg)
 		}
 	}
+	os.Args = newArgs
 }
 
 func main() {

--- a/tests/-Q-parse-meta-go-import.test.sh
+++ b/tests/-Q-parse-meta-go-import.test.sh
@@ -1,18 +1,46 @@
+_process=""
+
 :project "main.go" <<GO
 package main
 
-import foo "manul-test-foo.appspot.com/a/b"
+import foo "__blankd__/kovetskiy/manul-test-foo"
 
 func main() {
-    foo.Foo()
+    bar.A()
 }
 GO
 
-# NOTE: See https://github.com/hnakamur/google-app-engine-manul-test-foo
-# for source code of application manul-test-foo
-:lib "manul-test-foo.appspot.com/a/b"
+tests:put server <<SRV
+#!/bin/bash
 
-tests:ensure :manul -Q
+cat <<HTTP
+200 OK
+
+<meta name="go-import" content="localhost:60001/kovetskiy/manul-test-foo git https://github.com/kovetskiy/manul-test-foo" />
+HTTP
+SRV
+tests:ensure chmod +x $(tests:get-tmp-dir)/server
+
+
+:lib "github.com/kovetskiy/manul-test-foo"
+tests:ensure  mv \
+    $(tests:get-tmp-dir)/go/src/github.com \
+    $(tests:get-tmp-dir)/go/src/__blankd__
+
+tests:ensure blankd \
+    -l localhost:60001 \
+    -e $(tests:get-tmp-dir)/server \
+    -o $(tests:get-tmp-dir)/blankd.log \
+    --tls
+tests:value _process cat $(tests:get-stdout-file)
+:stop_blankd() {
+    if [[ "$_process" ]]; then
+        tests:eval kill "$_process"
+    fi
+}
+trap :stop_blankd EXIT
+
+tests:ensure :manul --integration-test --insecure-skip-verify -Q
 tests:assert-no-diff stdout <<VENDORS
-manul-test-foo.appspot.com/a/b
+localhost:60001/kovetskiy/manul-test-foo
 VENDORS

